src/Creator/AbstractProductCreator.php

```diff
@@ -75,6 +78,15 @@ protected function associateFeatureValues(int $productId, int $featureId, array
         }
     }
 
+    protected function associateStockMovements(int $productId, int $numberOfStockMovements): void
```

Comment:
Seems good to me but you didn't handle the stock creation for combinations?

It doesn't seem too much effort to add the feature, this method needs an extra `int $combinationId = 0` parameter and you can then call it in the `ProductCombinationCreator::generate` method this association methods (the CQRS command returns the list of generated combination IDs so you could loop on them)